vdr/src/contracts/auth/role_control.rs

```diff
@@ -201,10 +201,45 @@ pub mod test {
 
             assert_eq!(expected_transaction, transaction);
         }
+
+        #[async_std::test]
```

Comment:
bad test.   it does same steps as function itself to prepare expected data..

Reply:
Removed test since we have the same test above with expected data prepared